style.css

```diff
@@ -0,0 +1,247 @@
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 24px;
+}
+
+.prg {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #c8c8d4;
+}
+
+.p-div {
+  margin: 24px;
+}
+
+.h1 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 44px;
+  line-height: 72px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: #7d73fc;
+  margin-top: 56px;
+}
+
+.nav-list1 {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 24px;
+  gap: 8rem;
+}
+
+.hr1 {
+  height: 0;
+  left: 7px;
+  margin: 24px;
+  width: 327px;
+  right: 17px;
+  top: calc(50% - 0px / 2 + 0.5px);
+  text-align: center;
+  border: 1px solid #646369;
+}
+
+.lang {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  letter-spacing: 0.03em;
+  color: #9490ff;
+}
+
+.skills {
+  display: grid;
+  grid-template-columns: auto auto;
+  margin: 44px;
+}
+
+.tec {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #f5f5ff;
+}
+
+.h-li {
+  width: 98px;
+  font-size: 15px;
+  font-family: 'roboto', sans-serif;
+}
+
+.h-li a {
+  color: #fff;
+}
+
+.social-icon {
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.w-arrow {
+  display: flex;
+  justify-content: center;
+}
+
+.wk-arr {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+/* Card section */
+
+.r-work {
+  font-family: 'Roboto', sans-serif;
+  font-size: 40px;
+  line-height: 111%;
+  width: 221px;
+  margin-left: 24px;
+  color: #9490ff;
+}
+
+.img-p {
+  width: 327px;
+  height: 236px;
+  text-align: center;
+  margin-left: 24px;
+}
+
+.m-stories {
+  font-family: 'Roboto', sans-serif;
+  font-size: 32px;
+  font-weight: 500;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-top: 46px;
+  margin-bottom: 46px;
+  margin-left: 24px;
+}
+
+.c-prg {
+  height: 204px;
+  width: 327px;
+  left: 0;
+  top: 129px;
+  font-family: 'Inter', sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-left: 24px;
+}
+
+.c-list {
+  display: flex;
+  flex-direction: row;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: left;
+  gap: 1rem;
+  margin-left: 24px;
+}
+
+.c-d {
+  color: #555;
+  height: 40px;
+  width: 1px;
+  left: 130px;
+  top: 0;
+  border-radius: 0;
+  font-size: 24px;
+}
+
+.btn {
+  height: 58px;
+  width: 100px;
+  border-radius: 0;
+  padding: 12px;
+  background-color: #7d73fc;
+  margin-left: 24px;
+  margin-bottom: 40px;
+  font-size: 12px;
+  color: #fff;
+}
```

Comment:
- [ ] It is a requirement that the button interactions be added once there is a button on the page. Kindly add the button interactions as requested by the previous reviewer